projects/ngx-device-detector/src/lib/device-detector.service.ts

```diff
@@ -30,6 +31,9 @@ export class DeviceDetectorService {
   os_version = '';
   browser_version = '';
   reTree = new ReTree();
+  storeDesktop: boolean = null;
+  storeMobile: boolean = null;
```

Comment:
Why do we need these storeMobile etc variables?

Reply:
I removed it.